### PR TITLE
Ansible-playbook for openshift cluster setup

### PR DESCRIPTION
**Pull request by shashank855**
**What this PR does / why we need it**:
- Contains ansible-playbooks `create-openshift-cluster.yml` for openshift cluster setup.
- Includes a bash script `dependency.sh` which is required for all the necessary package installation before installing openshift.
- Contains a expect script for `oc login` command to execute.
- Contains a `inventory.sh` which is required for openshift installation. 

**Special notes for your reviewer**:
- `crate-openshift-cluster.yml` cotains tasks for: **creation ec2-instances**, **volumes**, **Setting up password less ssh on each nodes**, **installing dependency on each nodes**, **installing openshift**, **setting up the cluster for OpenEBS to deploy.**

### CODE REVIEW

File: k8s/aws/openshift-installer/create-openshift-cluster.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/prerequisites.yml -vv"'
+          async: 100
+          ignore_errors: yes
+
+        - name: Running openshift pre-requisites
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/prerequisites.yml -vv"'
+
+        - name: Installing Openshift cluster
+          shell: ssh -o StrictHostKeyChecking=no -i openshift.ppk centos@{{ master_dns.stdout }} 'sudo su -c "ansible-playbook -i /home/centos/inventory.yml /home/centos/openshift-ansible/playbooks/deploy_cluster.yml -vv"'
+
```
Comment (gprasath) [inline k8s/aws/openshift-installer/create-openshift-cluster.yml]: Give a comment why we execute the same task again

File: k8s/aws/openshift-installer/delete-instance.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  vars_files:
+    - vars.yml
+  
+  vars:
+    route_table_id: "{{ lookup('lines', 'grep route_table_id ./id.csv | cut -d, -f5 | cut -d: -f2') }}"
+    gateway_id: "{{ lookup('lines', 'grep gateway_id ./id.csv | cut -d, -f4 | cut -d: -f2') }}"
+    subnet_id: "{{ lookup('lines', 'grep subnet_id ./id.csv | cut -d, -f3 | cut -d: -f2') }}"
+    vpc_name: "{{ lookup('lines', 'grep vpc_name ./id.csv | cut -d, -f2 | cut -d: -f2') }}"
+    vpc_id: "{{ lookup('lines', 'grep vpc_id ./id.csv | cut -d, -f1 | cut -d: -f2') }}"
```
Comment (gprasath) [inline k8s/aws/openshift-installer/delete-instance.yml]: Can we move this to vars_file

File: k8s/aws/openshift-installer/create-openshift-cluster.yml (hunk 1 above)
Comment (shashank855) [reply k8s/aws/openshift-installer/create-openshift-cluster.yml]: @gprasath I have made the change.

File: k8s/aws/openshift-installer/delete-instance.yml (hunk 2 above)
Comment (shashank855) [reply k8s/aws/openshift-installer/delete-instance.yml]: @gprasath I have made the change.

**Pull request merged**